### PR TITLE
Feature/css mobile design

### PR DESCRIPTION
**Pull request by bhohnco**
## Type of Change
- [ ] JSX
- [ ] HTML
- [ ] JavaScript
- [X ] CSS
- [X] React
- [ ] React Router
- [ ] Cypress
- [ ] Apollo
- [ ] GraphQL
- [ ] Twilio API
- [ ] Refactor/Fix Bugs

## Please describe the change made
Mobile responsiveness has been add to the application 

## Please give a short description of the changes that were made below

- Fix:

- Feature:
All views now have a mobile and flow. 
main view, form, alert, current trip, add contacts, and bio 
needs some error/debugging tomorrow 

## How has this been tested?
- [ X] In Browser
- [ X] In Console
- [ ] Cypress
- [ ] Postman

### CODE REVIEW

File: src/Components/CurrentTrip/_currenttrip.scss [hunk 1]
```diff
@@ -70,3 +71,29 @@
...
   }
 }
+
+@media screen and (max-width: 360px) {
+  //.app-header {
+  //  font-size: 1em;
+  //  letter-spacing: 0.25em;
+  //}
+
+  .welcome-msg {
```
Comment (pcmueller) [inline src/Components/CurrentTrip/_currenttrip.scss:81]: Did adjusting the welcome message size help at all with the nav bar sizing?

File: src/Components/Alert/_alert.scss [hunk 2]
```diff
@@ -55,4 +55,13 @@
   font-size: 2.5rem;
   color:rgb(255, 255, 255);
   text-shadow: 0 1px 2px rgb(37, 34, 34);
+}
+
+@media screen and (max-width: 360px) {
```
Comment (pcmueller) [inline src/Components/Alert/_alert.scss:60]: Nice work, this display is looking great!

File: src/Components/TripETA/_trip-eta.scss [hunk 3]
```diff
@@ -46,4 +46,36 @@
...
+      margin:0.4em
+    }
+
+  .eta-message span:nth-of-type(2n) {
+    font-size: 1.15em;
+    font-weight: 500;
+    margin: 0.3em;
+  }
+
+  .begin-trip-btn {
```
Comment (pcmueller) [inline src/Components/TripETA/_trip-eta.scss:73]: I'm happy to take a look at button size responsiveness in the morning - just remind me which specifically we need to adjust.

**Review by pcmueller — APPROVED** [review]
Thanks so much for taking on responsiveness on your own - mobile views are looking great!

**Review by joemecha — APPROVED** [review]
Awesome. Looking forward to checkout out the result of these changes.

File: src/Components/AddTime/_addtime.scss [hunk 4]
```diff
@@ -78,3 +77,34 @@
   border-radius: 5px;
   width: 45%;
 }
+
+@media screen and (max-width: 360px) {
+
+  .timeout-message {
+    font-size: 1.3rem;
+
+    span:nth-of-type(1n) {
```
Comment (carolineshowscode) [inline src/Components/AddTime/_addtime.scss:86]: nice work utilizing nth child properties

**Review by carolineshowscode — APPROVED** [review]
NIce work!

**Pull request merged**